openwisp_controller/config/tests/test_controller.py

```diff
@@ -181,7 +181,7 @@ def test_get_cached_checksum(self):
         url = reverse("controller:device_checksum", args=[d.pk])
 
         with self.subTest("first request does not return value from cache"):
-            with self.assertNumQueries(4):
+            with self.assertNumQueries(3):
```

Comment:
Might have to check but do you know why the number of queries have been reduced here?

Reply:
The reduced count was due to prefetched organization settings when device was retrieved.